runtime/src/main/kotlin/kotlin/native/concurrent/Worker.kt

```diff
@@ -35,8 +35,10 @@ public inline class Worker @PublishedApi internal constructor(val id: Int) {
          * better to use non-blocking IO combined with more lightweight coroutines.
          *
          * @param errorReporting controls if an uncaught exceptions in the worker will be printed out
+         * @param name defines
          */
-        public fun start(errorReporting: Boolean = true): Worker = Worker(startInternal(errorReporting))
+        public fun start(errorReporting: Boolean = true, name: String? = null): Worker
```

Comment:
This will likely break binary compatibility between libraries and stdlib, which is undesirable for 1.3.60 EAP 2-3.